src/Model/Photo.php

```diff
@@ -150,7 +150,7 @@ public static function getPhoto(string $resourceid, int $scale = 0)
 
 		$uid = $r["uid"];
 
-		$accessible = $uid ? DI::pConfig()->get($uid, 'system', 'accessible-photos') : false;
+		$accessible = $uid ? (bool)DI::pConfig()->get($uid, 'system', 'accessible-photos', false) : false;
```

Comment:
Just FYI, the cast to boolean was unnecessary, the default value was enough to fix the error. Any set config value would have been automatically cast to boolean by the existence of the type-hint in `getPermissionsSQLByUserId`. Only `NULL` values can't be cast to the expected type.

Reply:
Ah okay. I wasn't totally sure about this.